helper.js

```diff
@@ -72,10 +70,8 @@ function addNodeInfos(node, id) {
         value.push(["Decomposition Block", node.decomBlock.replaceAll("\\n", "<br><br>")]);
         break;
       case "number of parent nodes":
```

Comment:
This case does not really do anything, so it can be removed, right? (also the next one)